docs/content/wasm.md

```diff
@@ -33,14 +33,24 @@ You can compile Rego policies into Wasm modules using the `opa build` subcommand
 
 For example, the `opa build` command below compiles the `example.rego` file into a
 Wasm module and packages it into an OPA bundle. The `wasm` target requires exactly
-one entrypoint (specified by `-e`).
+one entrypoint rule (specified by `-e`).
 
 ```bash
 opa build -t wasm -e example/allow example.rego
 ```
 
+The output of a Wasm module built this way contain the `result` of evaluating the
+entrypoint rule. For example:
+```json
+[
+    "result": <value of data.example.allow>
+]
```

Comment:
Nit: array of objects here:

```json
[
  {
    "result": <value of data.example.allow>
  }
]
```

Reply:
A little bit more detail could be useful after the example. Something like:

The output of policy evaluation is a set of variable assignments. The variable assignments specify values that satisfy the expressions in the policy query (i.e., if the variables in the query are replaced with the values from the assignments, all of the expressions in the query would be defined and not false.)

When policies are compiled into Wasm, the user provides the path of the policy decision that should be exposed by the Wasm module. The policy decision is _assigned_ to a variable named `result`. The policy decision can be ANY JSON value (boolean, string, object, etc.) but there will be at-most-one assignment. This means that callers should first check if the set of variable assignments is empty (indicating an undefined policy decision) otherwise they should select the `"result"` key out of the variable assignment set.

Reply:
Done